types/status-bar/README.md

```diff
@@ -1 +1 @@
-***NOTE:*** *This package is deprecated and new versions have been merged into [`@types/atom`](https://www.npmjs.com/package/@types/atom). The most recent version can be accessed as part of the `Atom.Services` namespace.*
+***NOTE:*** *This package is deprecated and new versions have been merged into [`@types/atom`](https://www.npmjs.com/package/@types/atom). Type definitions for the most recent version can be accessed from within the "atom/status-bar" module.*
```

Comment:
Maybe this package should be deleted from DefinitelyTyped and deprecated?

Reply:
It's a legacy definition that I have no part of really. It probably should be removed, but I don't really know how deprecation works here on DT with the types publisher. I can remove it with this commit if you'd like.

Reply:
It looks like the `status-bar` NPM package is something else entirely, so this should definitely be deprecated. #20727